packages/babel-preset-default/index.js

```diff
@@ -19,6 +19,7 @@ module.exports = function( api ) {
 
 		if ( isTestEnv ) {
 			opts.useBuiltIns = 'usage';
+			opts.corejs = 3;
```

Comment:
It looks like we enabled `corejs` v3 only for tests which is expected. Related discussion here: https://github.com/WordPress/gutenberg/pull/9171/files#r214595243